lms/templates/student_profile/badge_placeholder.underscore

```diff
@@ -0,0 +1,10 @@
+<div class="badge-display badge-placeholder">
+  <div class="badge-image-container">
+    <span class="accomplishment-placeholder" aria-hidden="true">
+  </div>
+  <div class="badge-details">
+    <div class="badge-name"><%- gettext('Earn Your Next Accomplishment') %></div>
+    <p class="badge-description"><%- gettext('Find a course and start learning towards a goal.') %></p>
```

Comment:
"Start working toward your next learning goal."
(Remove "Find a course" to avoid repeating the button text; and much as it hurts me to remove that "s", our style is the US "toward" rather than Cdn/UK "towards") :-)